docs/sdk/serverless-signing/zero-to-hero.md

```diff
@@ -0,0 +1,181 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Zero to Hero
+
+## Prerequisites
+
+- Familiarity with JavaScript
+
+## Overview
+In this section we're gonna cover an end-to-end example from minting a PKP through the ContractsSDK, setting its scope & signing a message with it inside a Lit Action.
+
+Installed the latest contracts-sdk on `cayenne`
```

Comment:
consider adding a 'step 1' or 'prerequisites' header here to set some context. Maybe also include a hyper link to the v3 repo?

Reply:
Added "Installation"